site.yml

```diff
@@ -133,7 +134,7 @@
     - role: keystone-setup
       keystone_configured: false
       tags: ['openstack', 'setup', 'keystone-setup']
-      when: not keystone_configured
+      #when: not keystone_configured
```

Comment:
this was commented out because it was getting in the way...not sure why we need to check for this.